sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/aad/webapp/AzureOAuthenticationFailureHandler.java

```diff
@@ -31,7 +31,7 @@ public AzureOAuthenticationFailureHandler() {
     public void onAuthenticationFailure(HttpServletRequest request,
                                         HttpServletResponse response,
                                         AuthenticationException exception) throws IOException, ServletException {
-
+        //Handle conditional access policy, step 2.
```

Comment:
1. It's better to add a space after `//`.
2. It's better to move it to line 46.